bitcoinj/0.15.6-rsk-3/README.md

```diff
@@ -0,0 +1,27 @@
+# BitcoinJ 0.15.6-rsk-3
+
+* Source: https://github.com/rsksmart/bitcoinj
+* Tag: `v0.15.6-rsk-3`
+
+## Build
+
+```
+$ docker build -t bitcoinj/0.15.6-rsk-3 .
+```
+
+## Verify
+
+```
+$ docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
+dea5d3f9e4cee70cde6fca8a26f9d4e975431c4b192ddade899fb0ab3bf34ecc  bitcoinj-core-0.15.6-rsk-3-sources.jar
+b1759d2bd9ff18b9631aa7eb57409fe67749d5d64739cff8b7cc0160bcc0f61a  bitcoinj-core-0.15.6-rsk-3.jar
+580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

Comment:
I got different results:

```
 docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
5c2e0ed68a57803c4a9d11d3889e4fdfd4775a8ad71f9d3f5db7e0896558e5eb  bitcoinj-core-0.15.6-rsk-3-sources.jar
8a720073381d052b888437d7123a39d45dbd2d008839507afec58fec3dded63e  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

May I be doing something wrong?

Reply:
same here, i got different results for both jars. Also different from natha's

```
docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
5597c66623cce82397fccf361b681cb6bfd7f7de0cf04cd620c3d1c020565b55  bitcoinj-core-0.15.6-rsk-3-sources.jar
8222d818b4491b99e775326adc01dd8af12fbde4040300fbcc70345dc167060f  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

Reply:
```
> docker run --rm bitcoinj/0.15.6-rsk-3 sh -c 'sha256sum * | grep -v javadoc.jar'
987a6fcacbdfc919c4c826f267c5803195d1770c96c852d7dc6f522ad8c2c7b9  bitcoinj-core-0.15.6-rsk-3-sources.jar
92ae400ba41e9c25a2a08e257d611f8fcedc3fc13baae40663856cb13860e7fa  bitcoinj-core-0.15.6-rsk-3.jar
580f70f8ec242c90d8141d2cdd501c1b9ee11b3c70972308e8c47b4f96acf3cf  bitcoinj-core-0.15.6-rsk-3.pom
```

Now, only the sources jar is the one which differs from your result.